contracts/MilestonePricing.sol

```diff
@@ -46,9 +46,7 @@ contract MilestonePricing is PricingStrategy, Ownable {
   /// @param _milestones uint[] milestones Pairs of (time, price)
   function MilestonePricing(uint[] _milestones) {
     // Need to have tuples, length check
-    if(_milestones.length % 2 == 1 || _milestones.length >= MAX_MILESTONE*2) {
-      throw;
-    }
+    require((_milestones.length % 2 == 0) && (_milestones.length < MAX_MILESTONE*2));
```

Comment:
Maybe two different require()s? They are dealing with the same data, though

Reply:
Since they are dealing with the same data, think better to use same require(). Have changed condition to _milestones.length <= MAX_MILESTONE * 2 similar to EthTranchePricing.sol.